.drone.yml

```diff
@@ -3,31 +3,35 @@ kind: pipeline
 name: default
 
 services:
-  - name: docker
-    image: docker:19.03.0-rc3-dind
-    privileged: true
-    command:
-      - --dns=8.8.8.8
-      - --dns=8.8.4.4
-      - --mtu=1440
-    volumes:
-      - name: docker-socket
-        path: /var/run
+- name: docker
```

Comment:
I ran all the YAML files in this repo through a YAML formatter that I'm hoping we can make part of the builds one day. Looks like this undoes that.